### DISCUSSION TITLE
Where to get your board ?

### DISCUSSION THREAD

**Discussion by drom89** [phil-barrett/grblHAL-teensy-4.x#34] [General]
Hello Phil,
is there any other way to order your board ? (JLCPCB or something like that ?)
Tindie isnt shipping to Czech Republic and i think shipping would cost same as the board :(

**Comment by phil-barrett** [comment]
I would consider shipping to the Czech Republic.  Shipping via USPS will be around $20 and could take several weeks.  I've been burned  shipping to some countries (lost 4 boards somewhere in South Africa, for example) so I tend to be fairly conservative.  Let me know if that's ok and I will turn it on for you.

For faster service, I have shipped via UPS a number of times internationally. It is very expensive and there are additional costs (brokerage fees, local taxes) that are hard to predict. That leads to unfortunate surprises and unhappy customers so I have stopped doing that in general.

**Comment by drom89** [comment]
I will contact you after christmas.

**Comment by heripriyanto** [comment]
Phil....I ask permission to order boards from jlcpcb. I live in
Indonesia, it seems impossible to buy at tindie considering shipping
costs, thank you.


Pada tanggal Sab, 4 Des 2021 03.21, phil-barrett ***@***.***>
menulis:
